compiler/luci/pass/src/FoldAddV2Pass.cpp

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci/Pass/FoldAddV2Pass.h"
+
+#include <luci/IR/CircleNodes.h>
+
+#include <iostream>
+namespace
+{
+
+bool same_static_shape(const luci::CircleNode *x, const luci::CircleNode *y)
```

Comment:
Can you apply following change?

```suggestion
bool same_static_shape(const luci::CircleConst *x, const luci::CircleConst *y)
```

Since `CircleConst` always has static shape, it seems more proper when considering function name :)

Reply:
> Since CircleConst always has static shape

If so, `same_shape` looks better for `CircleConst` (`static` is unnecessary).